smdebug/tensorflow/keras.py

```diff
@@ -947,14 +951,35 @@ def _save_layer_values(self, logs):
                 layer_name = layer_name.name
             elif isinstance(layer_name, bytes):
                 layer_name = str(layer_name, "utf-8")
+            layer_input_tensor_name = get_export_name_for_keras(str(layer_name), "input")
             if len(layer_input) == 1:
```

Comment:
nit: Do we still need this if block? Even if there is only one tensor, it might be better to keep the naming schema the same (and still include an index in the name).

Reply:
There's a performance improvement if we do not process intermediate layer values as lists.
And more often than not, they are single values.